File: application/ms-business/ms-adventures/src/main/java/com/wildadventures/msadventures/web/controller/dto/AdventureDto.java
package com.wildadventures.msadventures.web.controller.dto;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;
import java.util.List;

@Data
@Getter
@Setter
public class AdventureDto {
    private int id;
    private String name;
    private String description;
    private LocalDateTime startDate;
    private double price;
    private int maxParticipantNumber;
    private String categoryId;
    private List<AdventureImageDto> adventureImages;
}

File: application/front-end/wild-adventures/src/app/shared/models/dto/adventures/AdventureDTO.ts
import { AdventureImageDTO } from './AdventureImageDTO';

export class AdventureDTO {
    id: number;
    name: string;
    description: string;
    startDate: Date;
    price: number;
    maxParticipantNumber: number;
    categoryId: string;
    adventureImages: AdventureImageDTO[];

    constructor() {}
}
